charts/orchestrator/values.schema.json

```diff
@@ -405,6 +406,47 @@
                                 "username": "ARGOCD_USERNAME",
                                 "password": "ARGOCD_PASSWORD"
                             }]
+                        },
+                        "notifications_email": {
+                            "type": "object",
+                            "default": {},
+                            "title": "The notifications_email Schema",
+                            "required": [
+                                "hostname",
+                                "username",
+                                "password"
+                            ],
+                            "properties": {
```

Comment:
why sender and reply_to are not here ? (saw them below but I'm not sure what is the difference)

Reply:
they aren't sensitive information and can be provided via the helm chart as values.